packages/thirdweb/src/utils/encoding/to-bytes.ts

```diff
@@ -116,32 +75,13 @@ export type HexToBytesOpts = {
  * @utils
  */
 export function hexToBytes(hex_: Hex, opts: HexToBytesOpts = {}): Uint8Array {
-  let hex = hex_;
-  if (opts.size) {
-    assertSize(hex, { size: opts.size });
-    hex = padHex(hex, { dir: "right", size: opts.size });
-  }
-
-  let hexString = hex.slice(2) as string;
-  if (hexString.length % 2) {
-    hexString = `0${hexString}`;
-  }
-
-  const length = hexString.length / 2;
-  const bytes = new Uint8Array(length);
-  for (let index = 0, j = 0; index < length; index++) {
-    const nibbleLeft = charCodeToBase16(hexString.charCodeAt(j++));
-    const nibbleRight = charCodeToBase16(hexString.charCodeAt(j++));
-    if (nibbleLeft === undefined || nibbleRight === undefined) {
-      throw new Error(
-        `Invalid byte sequence ("${hexString[j - 2]}${
-          hexString[j - 1]
-        }" in "${hexString}").`,
-      );
+  const evenLengthHex = (() => {
+    if (hex_.length % 2 !== 0) {
+      return ox__Hex.padLeft(hex_, 2);
     }
-    bytes[index] = nibbleLeft * 16 + nibbleRight;
-  }
-  return bytes;
+    return hex_;
+  })();
```

Comment:
Made a change for Ox to handle this (which is the current Viem behavior). Try update Ox to `0.1.3`!